.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ on:
       - nixpkgs
   pull_request:
 env:
-  GO_VERSION: '1.22'
+  GO_VERSION: '1.21'
```

Comment:
We need go 1.22 for Kubernetes 1.30, which is scheduled for April 17.

Reply:
yea..

I'm not sure what to do here with runc not building on golang 1.22. Maybe we can see about runc relaxing the restriction on build as we are going to get it working with golang 1.22 eventually?

Reply:
opened https://github.com/kubernetes/kubernetes/issues/124168 as well as @saschagrunert and I brought up some slack threads with the release team

Reply:
Let's get this one in in the meanwhile.